llvm/include/llvm/TargetParser/PPCTargetParser.def

```diff
@@ -0,0 +1,80 @@
+#ifndef PPC_FEATURE
```

Comment:
we will support these feature in AIX OS soon.
can we change `PPC_FEATURE` to `PPC_LINUX_FEATURE`

and `PPC_CPU` to `PPC_LINUX_CPU` 

and when implement the __builtin_cpu_is and __builtin_cpu_supports in AIX OD

we can `PPC_AIX_FEATURE` and `PPC_AIX_CPU`